src/Settings/Builders/RunTest.hs

```diff
@@ -85,7 +84,10 @@ getTestArgs = do
     bindir          <- expr $ setBinaryDirectory (testCompiler args)
     compiler        <- expr $ setCompiler (testCompiler args)
     globalVerbosity <- shakeVerbosity <$> expr getShakeOptions 
-    let testOnlyArg  = case testOnly args of
+    let configFileArg= case testConfigFile args of
+                           Just filepath -> ["--config-file=" ++ filepath]
+                           Nothing       -> ["--config-file=testsuite/config/ghc"]
```

Comment:
I'd recommend just using testsuite/config/ghc as the default value, as pointed out in my other comment. If an arg is given, use it, otherwise use the default value.

Reply:
Resolved